packaging/hudi-utilities-bundle/pom.xml

```diff
@@ -122,6 +122,8 @@
                   <include>com.github.davidmoten:guava-mini</include>
                   <include>com.github.davidmoten:hilbert-curve</include>
                   <include>com.github.ben-manes.caffeine:caffeine</include>
+                  <!-- SPARK-43489 Spark 3.5+ has marked protobuf as provided -->
+                  <include>com.google.protobuf:protobuf-java</include>
                   <include>com.twitter:bijection-avro_${scala.binary.version}</include>
```

Comment:
This is needed otherwise deltastreamer would fail due to
```
Exception in thread "main" java.lang.NoClassDefFoundError: com/google/protobuf/Message
    at java.lang.Class.getDeclaredMethods0(Native Method)
    at java.lang.Class.privateGetDeclaredMethods(Class.java:2729)